lib/lib.rs

```diff
@@ -243,6 +246,121 @@ impl Parser {
   }
 }
 
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct EszipError {
+  message: String,
+  specifier: Option<String>,
+  line: Option<usize>,
+  column: Option<usize>,
```

Comment:
If there's a line, then there's a column so these could be combined to be behind a single `Option` to make it easier for consumers.